yt/frontends/ramses/data_structures.py

```diff
@@ -40,9 +40,12 @@ class RAMSESFileSanitizer:
     info_fname = None  # Path | None: path to the info file
     group_name = None  # str | None: name of the first group folder (if any)
 
-    def __init__(self, filename):
+    def __init__(self, filename, preserve_symlinks=False):
+
+        filename = Path(filename)
         # Resolve so that it works with symlinks
-        filename = Path(filename).resolve()
+        if not preserve_symlinks:
+            filename = Path(filename).resolve()
```

Comment:
Is there any situation when using `.resolve()` would be a bad thing? If no, how about we skip all the hassle of adding additional kwargs here?

Reply:
The default was to use `.resolve()` so I assumed that there was a good reason for it to be there. I found that _removing_ it fixed my issue, but I didn't want to lead to backward-incompatible code without a good reason.

Reply:
Oops, sorry I misread it.